From: Daniel Golle <daniel@makrotopia.org>
Date: Sun, 27 Dec 2020 15:03:22 +0000
Subject: [PATCH] use correct printf format for size_t

pdkim.c: In function 'check_bare_ed25519_pubkey':
pdkim.c:1355:60: warning: format '%lu' expects argument of type 'long unsigned int', but argument 2 has type 'size_t' {aka 'unsigned int'} [-Wformat=]
   DEBUG(D_acl) debug_printf("DKIM: unexpected pubkey len %lu\n", p->key.len);
                                                             ~~^     ~~~~~~~~~~
                                                                                                                       %u
Signed-off-by: Daniel Golle <daniel@makrotopia.org>
---
--- a/src/pdkim/pdkim.c
+++ b/src/pdkim/pdkim.c
@@ -1352,7 +1352,7 @@ check_bare_ed25519_pubkey(pdkim_pubkey *
 int excess = p->key.len - 32;
 if (excess > 0)
   {
-  DEBUG(D_acl) debug_printf("DKIM: unexpected pubkey len %lu\n", p->key.len);
+  DEBUG(D_acl) debug_printf("DKIM: unexpected pubkey len %zu\n", p->key.len);
   p->key.data += excess; p->key.len = 32;
   }
 }
